test/selenium/src/lib/page/widget/info_widget.py

```diff
@@ -462,20 +515,19 @@ def __init__(self, driver):
 class Controls(InfoWidget):
   """Model for Control object Info pages and Info panels."""
   # pylint: disable=too-many-instance-attributes
+  # pylint: disable=unbalanced-tuple-unpacking
```

Comment:
It's not a good practice to mute this pylint warning

Reply:
Done.